testLatestAkkaSnapshot.sh

```diff
@@ -1,7 +1,5 @@
-VERSION=$(curl https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -o '2.6\.[0-9]*+[0-9]*-[0-9a-z]*' | tail -n 1)
+VERSION=$(curl https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -o '2.6\.[0-9]*+[0-9]*-[0-9a-z]*' | sort -V | tail -n 1)
```

Comment:
TIL about -V nice

They come back in order from the listing so this was already working AFAIK